.circleci/config.yml

```diff
@@ -109,6 +141,14 @@ jobs:
             kubectl rollout restart statefulset nightly-dtl --namespace nightly
             kubectl rollout restart deployment nightly-gas-oracle --namespace nightly
             kubectl rollout restart deployment edge-proxyd --namespace nightly
+  notify:
```

Comment:
Did you mean to leave in this dummy job?

Reply:
Yep, it's an artifact of how CircleCI works :(